docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`1726` Fixed COMP zero price warnings on Compound history events for COMP claimed.
```

Comment:
Again, this is not a commit log. This is a changelog. User cares about what affects them.

So more like:

When querying compound history for COMP claimed around the start of COMP issuance, zero price warnings should no longer be emitted.